docs/docs/genai/eval-monitor/scorers/llm-judge/guidelines.mdx

```diff
@@ -0,0 +1,153 @@
+import { APILink } from "@site/src/components/APILink";
+import ImageBox from "@site/src/components/ImageBox";
+import TileCard from '@site/src/components/TileCard';
+import TilesGrid from '@site/src/components/TilesGrid';
+import { Bot, GitBranch, MessageSquare } from "lucide-react";
+
+# Create Custom LLM Scorers
+
+While MLflow's predefined LLM judge scorers offer excellent starting points for common quality dimensions in simpler applications, you'll need to create custom LLM judges as your application becomes more complex and to tune your evaluation criteria to meet the specific, nuanced business requirements of your use case and align with your domain expert's judgment. MLflow provides robust and flexible ways to create **custom LLM judges** tailored to these unique requirements.
+
+## Guidelines (_we suggest starting here_)
```

Comment:
Do we have a dedicate page for guidelines since we recommend users to start from this scorer?